README.md

```diff
@@ -94,6 +91,87 @@ Copied 🐍 💪 🎉 💓 🔥 to clipboard
 
 After sourcing your .zshrc you can now get emojis copied to your clipboard fast using `emo bicep`, `emo tada` etc.
 
+### Preferred emojis
```

Comment:
Started here!

Nice for a new article: _Documentation-driven development in practice_